nncf/quantization/passes.py

```diff
@@ -116,3 +118,21 @@ def filter_constant_nodes(
     constant_nodes = [node for node in nncf_graph.get_all_nodes() if node not in visited_nodes]
     nncf_graph.remove_nodes_from(constant_nodes)
     return nncf_graph
+
+
+TModel = TypeVar("TModel")
+
+
+def insert_null_biases_pass(model: TModel) -> TModel:
+    """
+    This pass finds and inserts zero biases to the given model for the layers that should have it.
+
+    :param model: Model instance.
+    :return: Updated Model instance with zero biases
+    """
+    model_backend = get_backend(model)
+    if model_backend == BackendType.OPENVINO:
+        from nncf.openvino.graph.model_utils import insert_null_biases
+
+        return insert_null_biases(model)
+    return model
```

Comment:
I would raise an error if backend is not supported

Reply:
I did it intentionally because of this logic
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/quantization/algorithms/bias_correction/onnx_backend.py#L132-L134

Reply:
That function looks really strange. It does nothing

Reply:
Let's discuss it offline